docs/visualize/vega.asciidoc

```diff
@@ -16,21 +16,20 @@ NOTE: In Vega it is possible to load data dynamically, e.g. by setting signals a
 [[vega-quick-demo]]
 === Getting Started with Vega
 
-Follow these steps to create your first Vega visualization.
-
-. In Kibana, choose Visualize, and add Vega visualization. You should see a default graph.
-. Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
-`square`, ... (see
-https://vega.github.io/vega-lite/docs/mark.html#mark-def[docs])
-. Try other https://vega.github.io/vega/examples/[Vega] or
-https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. You
-may need to make URLs absolute, e.g. replace
+* Go to *Visualize*, click *+*, and select *Vega* to see an example graph.
+* Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
+or `square`. Check out the 
+https://vega.github.io/vega-lite/docs/mark.html#mark-def[Vega docs] for more information.
+* Explore other available https://vega.github.io/vega/examples/[Vega] or
+https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. 
+Note: You might need to make URLs absolute, for example, replace
 `"url": "data/world-110m.json"` with
-`"url": "https://vega.github.io/editor/data/world-110m.json"`. (see
-link:#Using%20Vega%20and%20VegaLite%20examples[notes below])
-. Using https://www.npmjs.com/package/makelogs[makelogs util], generate
-some logstash data and try link:public/examples/logstash[logstash
-examples].  *(Do not use makelogs on a production cluster.)*
+`"url": "https://vega.github.io/editor/data/world-110m.json"`. 
+See <<vega-using-vega-and-vegalite-examples, Vega and VegaLite examples>>.
+* You can generate some Logstash data to experiment with using https://www.npmjs.com/package/makelogs[makelogs util]. 
```

Comment:
I think we may want to encourage users to use the new built-in datasets now that they are available (e.g. flight data, etc).  Logstash sample data is much more complex to generate.  Some Vega examples come with those data sets (note to self - add more vega examples to those datasets)

Reply:
Thanks @nyurik - of the three sample data sets, is there any one in particular with Vega examples I should point to?